tests/onnx/benchmarking/object_detection_segmentation/onnx_models_configs/fcn-resnet50-12.yml

```diff
@@ -14,6 +19,7 @@ models:
           has_background: True
           sort_annotations: True
           semantic_only: True
+          convert_mask: False
```

Comment:
Will this change affect the final accuracy?

Reply:
No, this is needed to not overwrite the existing masks